6-magic.md

```diff
@@ -2,12 +2,14 @@
 
 <!-- MarkdownTOC -->
 
-- Allocated spells
+- Allocating spells
```

Comment:
@Yther You actually can't edit these markdown TOC sections. They are auto generated by my code editor. We can remove them at a later date, but atm they provide a quick way of understanding what's in a file

Reply:
That's interesting, alright then! I was wondering why the titles of the magic section all say D6 after them.